agent/src/main/java/com/cloud/agent/properties/AgentProperties.java

```diff
@@ -69,6 +69,13 @@
      */
     public static final Property<Integer> VM_MEMBALLOON_STATS_PERIOD = new Property<>("vm.memballoon.stats.period", 0);
 
+    /**
+     * The number of iothreads
+     * Data type: Integer.<br>
+     * Default value: <code>0</code>
```

Comment:
This says default is 0, but the line below (Java code) shows that we set it to 1 by default.